### PR TITLE
docs: Hubspot stream `labels`

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
- User got tripped up with object `label` vs object `name`. https://github.com/airbytehq/oncall/issues/6190

## How
- Added tip in troubleshooting
- Cleaned up the docs while I was here

### CODE REVIEW

File: docs/integrations/sources/hubspot.md [hunk 1]
```diff
@@ -284,7 +276,9 @@ The connector is restricted by normal HubSpot [rate limitations](https://legacyd
   }
   ```
 
-- **Unnesting top level properties**: Since version 1.5.0, in order to not make the users query their destinations for complicated json fields, we duplicate most of nested data as top level fields.
+- **Hubspot object labels** In Hubspot, a label can be applied to a stream that differs from the original API name of the stream. Hubspot's UI shows the label of the stream, whereas Airbyte shows the name of the stream. If you are having issues seeing a particular stream your user should have access to, search for the `name` of the Hubspot object instead.
```
Comment (aldogonzalez8) [inline docs/integrations/sources/hubspot.md:279]: awesome

**Review by aldogonzalez8 — APPROVED** [review]
APPROVED

**Comment by aldogonzalez8** [comment]
There was a problem with unit tests unrelated to your documentation update; fix was merged [here](https://github.com/airbytehq/airbyte/pull/44197). It should pass now.

**Comment by aldogonzalez8** [comment]
You are all set. You can merge when you are ready.

**Comment by nataliekwong** [comment]
Thank you!!

**Pull request merged**